### PR TITLE
✨Add outcome endpoint

### PR DESCRIPTION
**Pull request by parimalak**
Added outcome resources and schemas
Updated tests for outcome resource

### CODE REVIEW

File: tests/outcome/test_outcome_resources.py [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+        self.assertEqual(kwargs['kf_id'], outcome['kf_id'])
+        
+        # Fields that should be updated w values
+        self.assertEqual(body['vital_status'],
+                         outcome['vital_status'])
+
+
+    def test_patch_bad_input(self):
+        """
+        Test updating an existing participant with invalid input
```
Comment (dankolbman) [inline tests/outcome/test_outcome_resources.py]: Needs to be updated.

File: tests/outcome/test_outcome_resources.py [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+from dataservice.api.outcome.models import Outcome
+from dataservice.api.participant.models import Participant
+from dataservice.api.study.models import Study
+from tests.utils import FlaskTestCase
+
+OUTCOMES_URL = 'api.outcomes'
+OUTCOMES_LIST_URL = 'api.outcomes_list'
+
+
+class OutcomeTest(FlaskTestCase):
```
Comment (znatty22) [inline tests/outcome/test_outcome_resources.py:16]: We should merge #84 first and then you can probably remove a lot of these tests (bad input, missing req params) and put them in test_api.py.

**Review by dankolbman — APPROVED** [review]
Looks good!

File: tests/test_api.py [hunk 3]
```diff
@@ -105,6 +112,7 @@ def test_status_format(self, client, endpoint, method):
         ('/samples', 'PATCH', ['created_at', 'modified_at']),
         ('/studies', 'POST', ['created_at', 'modified_at']),
         ('/studies', 'PATCH', ['created_at', 'modified_at']),
+        ('/outcomes', 'PATCH', ['created_at', 'modified_at']),
```
Comment (znatty22) [inline tests/test_api.py]: What about `('/outcomes', 'POST', ['created_at', 'modified_at'])`

**Review by znatty22 — COMMENTED** [review]
I think you're missing a relations test for `outcome` in test_api.py. In:

```python
def test_relations(self, client, entities, resource, field)
```

File: tests/test_api.py (hunk 3 above)
Comment (parimalak) [reply tests/test_api.py]: 👍

**Review by znatty22 — APPROVED** [review]
👍

**Pull request merged**